src/CONST.ts

```diff
@@ -3607,6 +3609,272 @@ const CONST = {
+                    message:
+                        'Here’s how to set up categories:\n' +
+                        '\n' +
+                        '1. Click your profile picture.\n' +
+                        '2. Go to <strong>Workspaces</strong> > [your workspace].\n' +
+                        '3. Click <strong>Categories</strong>.\n' +
+                        '4. Enable and disable default categories.\n' +
+                        '5. Click <strong>Add categories</strong> to make your own.\n' +
+                        '\n' +
+                        'For more controls like requiring a category for every expense, click Settings.',
+                },
+                {
+                    type: 'addExpenseApprovals',
+                    autoCompleted: false,
+                    title: 'Add expense approvals',
+                    subtitle: 'Add expense approvals to review your team’s spend and keep it under control.',
+                    message:
+                        'Here’s how to add expense approvals:\n' +
+                        '\n' +
+                        '1. Click your profile picture.\n' +
+                        '2. Go to Workspaces > [your workspace].\n' +
+                        '3. Click <strong>More features</strong>.\n' +
+                        '4. Enable <strong>Workflows</strong>.\n' +
+                        '5. In Workflows, enable <strong>Add approvals</strong>.\n' +
+                        '\n' +
+                        'You’ll be set as the expense approver. You can change this to any admin once you invite your team.',
+                },
+                {
+                    type: 'inviteTeam',
+                    autoCompleted: false,
+                    title: 'Invite your team',
+                    subtitle: 'Invite your team to Expensify so they can start tracking expenses today.',
+                    message:
+                        'Here’s how to invite your team:\n' +
+                        '\n' +
+                        '1. Click your profile picture.\n' +
+                        '2. Go to <strong>Workspaces</strong> > [your workspace].\n' +
+                        '3. Click <strong>Members</strong> > <strong>Invite member</strong>.\n' +
+                        '4. Enter emails or phone numbers. \n' +
+                        '5. Add an invite message if you want.\n' +
+                        '\n' +
+                        'That’s it! Happy expensing :)',
+                },
+            ],
+        },
+        [onboardingChoices.PERSONAL_SPEND]: {
+            message: 'Here’s how to track your spend in a few clicks.',
+            video: {
+                url: `${CLOUDFRONT_URL}/videos/intro-1280.mp4`,
+                thumbnailUrl: `${CLOUDFRONT_URL}/images/expensify__favicon.png`,
+                duration: 55,
+                width: 1280,
+                height: 960,
+            },
+            tasks: [
+                {
```

Comment:
We did not include `selfGuidedTourTask` in the tasks if the tour purpose is `combinedTrackSubmitOnboardingChoices.PERSONAL_SPEND` leading to this [issue](https://github.com/Expensify/App/issues/52985).